zerver/tests/test_signup.py

```diff
@@ -2520,11 +2584,8 @@ def test_signup_two_confirmation_links(self) -> None:
              'terms': True,
              'full_name': "New Guy",
              'from_confirmation': '1'})
-        # We should get redirected back to the login page.
-        expected_url = ('/accounts/login/' + '?email=' +
-                        urllib.parse.quote_plus(email))
-        self.assertEqual(result.status_code, 302)
-        self.assertEqual(result["Location"], expected_url)
+        # Error page should be displayed
+        self.assertEqual(result.status_code, 200)
```

Comment:
we need to assert error message content being in the response for this to actually test that an error page is being shown, as opposed to something else